src/index.css

```diff
@@ -73,60 +123,92 @@ body {
     animation-delay: 500ms
 }
 
-.episode__star {
+.tv-show-season__title {
+    font-size: 20px;
+}
+
+.tv-show-season__title :hover, .tv-show-season__title a:hover {
+    text-decoration: none;
+}
+
+.tv-show-episode {
+    border-top: #e3e3e3 solid 1px;
+    margin-bottom: 20px;
+    overflow: hidden;
+}
+
+.tv-show-episode__cover {
     float: left;
-    margin-right: 10px;
+    width: 250px;
+    min-height: 140px;
+    margin: 10px;
 }
 
-.favourite-star {
-    width: 25px;
+.tv-show-episode__cover img {
+    width: 250px;
 }
 
-.search-btn {
-    position: relative;
+.tv-show-episode__summary {
+    clear: left;
+    margin: 10px;
 }
 
-.search-btn__clear {
-    position: absolute;
-    right: 10px;
-    top: 0;
-    bottom: 0;
-    height: 20px;
-    margin: auto;
+.episode-info {
+    float: left;
+    padding-left: 10px;
+}
+
+.episode-info__title {
+    margin-top: 10px;
     font-size: 20px;
-    cursor: pointer;
-    color: #ccc;
 }
 
-.tv-show-seasons {
+.episode-info__item {
+    font-size: 14px;
+}
 
+/*noinspection CssUnusedSymbol*/
+.dv-star-rating-star {
+    font-size: 25px;
+    color: rgb(255, 180, 0) !important;
 }
 
-.tv-show-season {
+/*noinspection CssUnusedSymbol*/
+.favourite-icon {
+    color: #333;
+}
 
+/*noinspection CssUnusedSymbol*/
+.favourite-icon--filled {
+    color: #E10028;
 }
 
-.tv-show-episodes {
+/*noinspection CssUnusedSymbol*/
+.favourite-icon--filled:before {
+    content: "\f004";
+}
 
+/*noinspection CssUnusedSymbol*/
+.favourite-icon--unfilled:before {
+    content: "\f08a";
 }
 
-.tv-show-episode {
+.schedule-episode {
     border: #e3e3e3 solid 1px;
```

Comment:
border zapisuje sie w takiej kolejności:
`border: <width> <style> <color>` w sumie nie wiem czemu to działa... :D ale zmień na 1px solid #e3e3e3 bo tak to czyta i zpisuje większość programistów